# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by scripts/Create-CopyPatch.
# 
# T2 SDE: package/*/qt5/hotfix-gcc11.patch
# Copyright (C) 2021 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- qt-everywhere-src-5.15.2/qtbase/src/corelib/global/qglobal.h.vanilla	2021-06-27 17:09:09.583787984 +0200
+++ qt-everywhere-src-5.15.2/qtbase/src/corelib/global/qglobal.h	2021-06-27 17:35:00.992695513 +0200
@@ -45,6 +45,7 @@
 #  include <type_traits>
 #  include <cstddef>
 #  include <utility>
+#  include <limits>
 #endif
 #ifndef __ASSEMBLER__
 #  include <assert.h>
